compiler/one-cmds/onecc.template.cfg

```diff
@@ -1,29 +1,92 @@
+; To activate a step (or task),
+; set True for the step in [onecc] section and fill options in the corresponding section
 [onecc]
-one-import-tf=True
+; neural network model to circle
+one-import-tf=False
 one-import-tflite=False
 one-import-bcq=False
 one-import-onnx=False
-one-optimize=True
+; circle to circle with optimization
+one-optimize=False
+; circle to circle with quantization
 one-quantize=False
+; partition circle
 one-partition=False
-one-pack=True
+; package circle and metadata into nnpackage
+one-pack=False
+; generate code for backend
 one-codegen=False
+; profile
 one-profile=False
 
 [one-import-tf]
-input_path=/path/to/inception_v3.pb
-output_path=inception_v3.circle
-input_arrays=input
-input_shapes=1,299,299,3
-output_arrays=InceptionV3/Predictions/Reshape_1
-converter_version=v1
-model_format=graph_def
+; mandatory
+input_path=                 ; pb file
+output_path=                ; circle file
+; optional but mandatory for converter_version=v1
+input_arrays=               ; input tensor names
+input_shapes=               ; eg) 1,299,299,3
+output_arrays=              ; output tensor names
```

Comment:
From the point of user, as I assume most users will be using TF2.x, these three options won't be necessary. So how about moving them below `model_format=` item?

Reply:
Good point. In addition, I'll change `converter_version=v2` as default.